keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/README.md

```diff
@@ -647,8 +590,8 @@ Recall that end users cannot usually observe component logs or access metrics.
 - [ ] API .status
   - Condition name: 
   - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+- [x] Other (treat as last resort)
+  - Details: Provide logical cpu allocation distribution across physical cores and also the cpu cache metrics from ecosystem.
```

Comment:
resource managers are unfortunately poorly observable. The only ways we have fall in the
`Inspect the kubelet configuration of the nodes: check feature gate and usage of the new option.`
I think is good enough (and surely not worse) than all other changes in this area. We can think about metrics later in the graduation stage.

Reply:
Thanks for the suggestion. Let me update it to what you suggested and I think this is feasible.